### DISCUSSION TITLE
Torch version to run on GPU

### DISCUSSION THREAD

**Discussion by jobi12Isabel** [lava-nc/lava-dl#199] [General]
Hello,
I am trying to train my model on GPU. I am getting some issues due to torch version.
Which torch version should I install and which CUDA version am I supposed to have?
Thanks

**Comment by mgkwill** [comment]
> Hello,
> 
> I am trying to train my model on GPU. I am getting some issues due to torch version.
> 
> Which torch version should I install and which CUDA version am I supposed to have?
> 
> Thanks

If you use the poetry install method listed in the readme, the torch version installed is '1.13.1'. 

A google of 'torch "1.13.1" CUDA version' mentions CUDA 11.6 and 11.7, see https://pytorch.org/blog/PyTorch-1.13-release/

Have you tried either of those versions?
